src/legacy/ui/ui_bundles/ui_bundles_controller.js

```diff
@@ -168,7 +170,11 @@ export class UiBundlesController {
   }
 
   getCacheDirectory(...subPath) {
-    return this.resolvePath('../.cache', this.hashBundleEntries(), ...subPath);
+    return this.resolvePath(
+      '../../built_assets/.ui_bundles_cache',
```

Comment:
@spalger what do u think about that idea of changing the ui bundles cache directory name from `.cache` to `.ui_bundles_cache` ?

Reply:
Well, the point of the cache directory was to support named sub-directories, so I'd rather do `.cache/ui_bundles`

Reply:
👍